priority-queue-sf/PriorityQueue.Fabric/ApplicationParameters/Cloud.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Application xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="fabric:/PriorityQueue.Fabric" xmlns="http://schemas.microsoft.com/2011/01/fabric">
+  <Parameters>
+    <Parameter Name="HighFabric_InstanceCount" Value="-1" />
```

Comment:
There is no weighting of instance counts by priority. I see that there is weighting in the Local.5Node.xml and would expect weighting here too.